cmake/modules/BuildRocksDB.cmake

```diff
@@ -1,4 +1,5 @@
 function(build_rocksdb)
+  find_program (MAKE_EXECUTABLE NAMES gmake make)
```

Comment:
i don't think we need to "find_program()" here. because we don't depend on gmake or make to build rocksdb.